packages/dashboard/src/web3/context.ts

```diff
@@ -70,6 +70,7 @@ export const Web3Context: Web3ProviderType = {
     ),
     getAssetsList: PluginServices.Wallet.getAssetsList,
     getAssetsListNFT: PluginServices.Wallet.getAssetsListNFT,
+    getDomainsList: PluginServices.Wallet.getDomainsByTwitterId,
```

Comment:
`getENSDomains`?

Reply:
To keep it the same style as other RPC methods.